### PR TITLE
gh: Add workflow to build and push

### PR DESCRIPTION
**Pull request by fabiand**
Move building and pushing to the server side.

### CODE REVIEW

File: examples/fedora.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ spec:
         - sleep
         - "6000"
         image: >-
-            quay.io/openshift-virtualization/wasp-agent:latest
+            quay.io/fdeutsch/wasp-operator-prototype:latest
```
Comment (enp0s3) [inline examples/fedora.yaml:26]: nit: private registry URL

File: examples/static.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ spec:
           - -c
           - "dd if=/dev/urandom of=/dev/shm/rnd bs=1M count=60 ; sleep inf"
         image: >-
-            quay.io/openshift-virtualization/wasp-agent:latest
+            quay.io/fdeutsch/wasp-operator-prototype:latest
```
Comment (enp0s3) [inline examples/static.yaml:27]: same here

File: examples/stress.yaml [hunk 3]
```diff
@@ -33,7 +33,7 @@ spec:
           - -c
           - "sleep $(( RANDOM % 30 )) ; stress -v -m 1 --vm-bytes 1G --vm-hang 3"
         image: >-
-            quay.io/openshift-virtualization/wasp-agent:latest
+            quay.io/fdeutsch/wasp-operator-prototype:latest
```
Comment (enp0s3) [inline examples/stress.yaml:36]: same here

File: manifests/ds.yaml [hunk 4]
```diff
@@ -25,7 +25,7 @@ spec:
       containers:
         - name: wasp-agent
           image: >-
-            quay.io/openshift-virtualization/wasp-agent:latest
+            quay.io/fdeutsch/wasp-operator-prototype:latest
```
Comment (enp0s3) [inline manifests/ds.yaml:28]: same here

File: to.sh [hunk 5]
```diff
@@ -1,4 +1,4 @@
-IMG_REPO=quay.io/openshift-virtualization/wasp-agent
+IMG_REPO=quay.io/fdeutsch/wasp-operator-prototype
```
Comment (enp0s3) [inline to.sh:1]: same here

**Review by enp0s3 — COMMENTED** [review]
@fabiand Thanks! I have a small q below

File: examples/fedora.yaml (hunk 1 above)
Comment (fabiand) [reply examples/fedora.yaml:26]: OH OH OH, my bad :) Fixing

**Comment by fabiand** [comment]
Fixed all known issues, merging.

**Pull request merged**